src/plugins/data/public/actions/filters/create_filters_from_range_select.ts

```diff
@@ -19,12 +19,20 @@
 
 import { last } from 'lodash';
 import moment from 'moment';
+import { Datatable } from 'src/plugins/expressions';
 import { esFilters, IFieldType, RangeFilterParams } from '../../../public';
 import { getIndexPatterns, getSearchService } from '../../../public/services';
-import { RangeSelectContext } from '../../../../embeddable/public';
 import { AggConfigSerialized } from '../../../common/search/aggs';
 
-export async function createFiltersFromRangeSelectAction(event: RangeSelectContext['data']) {
```

Comment:
Rather than selecting the `data` property from `RangeSelectContext`, I've copied those pieces over here.